test/extensions/quic_listeners/quiche/integration/quic_http_integration_test.cc

```diff
@@ -290,13 +290,12 @@ TEST_P(QuicHttpIntegrationTest, MultipleQuicListenersWithBPF) {
   set_reuse_port_ = true;
   initialize();
   std::vector<IntegrationCodecClientPtr> codec_clients;
-  quic::QuicCryptoClientConfig::CachedState* cached = crypto_config_.LookupOrCreate(server_id_);
   for (size_t i = 1; i <= concurrency_; ++i) {
     // The BPF filter looks at the 1st word of connection id in the packet
     // header. And currently all QUIC versions support 8 bytes connection id. So
     // create connections with the first 4 bytes of connection id different from each
     // other so they should be evenly distributed.
```

Comment:
btw, we have change the BPF internally to inspect the 4 bytes starting from byte 1 (0-based). You may want to do the same here.

Reply:
ACK. I will make another change accordingly. What's the reason behind the change?